analyzers/src/SonarAnalyzer.CSharp/Extensions/SyntaxNodeExtensions.cs

```diff
@@ -22,7 +22,7 @@
 
 namespace SonarAnalyzer.Extensions
 {
-    internal static partial class SyntaxNodeExtensions
+    public static partial class SyntaxNodeExtensions
```

Comment:
I had to make it public so I could use it in my rule.
Is there a better approach? Maybe `InternalVisibleTo`?

Reply:
Public is better than internals visible to